src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,16 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+
+export const CommentList = ({ com }) => (
+  <>
+    <p>
+      {com.name}
+      <>  </>
+      by
+      <>  </>
+      <a href={`mailto:${com.email}`}>{com.email}</a>
+    </p>
+    <br />
+    <p>{com.body}</p>
```

Comment:
It looks like CommentInfo component but you called it CommentList and you also remove all CommentList styles